test/unit/DetectChanges/BCBreak/ClassBased/TraitUsePropertyRemovedTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
```

Comment:
There should be a pre-existing unit test for this: can you check? In theory, we just need to improve that

Reply:
I didn't find a class based test with a trait use, also changing the message reported didn't cause any failing tests so I expected that there was no test in the passed which covered this behaviour

Reply:
The class is https://github.com/Roave/BackwardCompatibilityCheck/blob/2eae3f72084f0bdc59847e486581f7882ba6a88c/test/unit/Formatter/ReflectionPropertyNameTest.php#L22 - the tests generally use a `StringSourceLocator` to make code snippets small and isolated :)

Reply:
I've added some testcases to that test